rampwf/workflows/image_classifier.py

```diff
@@ -42,18 +79,22 @@ def train_submission(self, module_path, folder_X_array, y_array,
         image_preprocessor = imp.load_source(
             '', submitted_image_preprocessor_file)
         transform_img = image_preprocessor.transform
+        has_transform_test = hasattr(image_preprocessor, 'transform_test')
+        transform_test_img = image_preprocessor.transform_test if has_transform_test else transform_img
```

Comment:
I think you can also do `transform_test_img = getattr(image_preprocessor, 'transform_test', transform_img)`